src/components/SellerTag.js

```diff
@@ -1,36 +1,95 @@
 import styled from 'styled-components';
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import { SVG } from './svg';
 
-const SellerTag = props => {
+const tagList = [
```

Comment:
- 상수는 대문자 스네이크 케이스로 표기해주세요!
- 아래 코드 참고해주세요!
```suggestion
const TAG_LIST = [
```
- 그리고 상수의 위치는 컴포넌트 아래, 스타일 컴포넌트 위로 위치시켜 주세요!
```jsx
const SellerTag = () => { ...  // 코드 생략}

export default SellerTag;

const TAG_LIST = [ ... ];

const SampleStyledComponent = styled.div`
  ...
`;
```

Reply:
SellerTag는 대웅님이 작성하신거여서 함께 수정했습니다!